lib/classes/InventoryFolder.ts

```diff
@@ -80,6 +80,24 @@ export class InventoryFolder
         return children;
     }
 
+    getChildFoldersRecursive(): InventoryFolder[]
+    {
+        const children: InventoryFolder[] = [];
+        const toBrowse: UUID[] = [this.folderID];
+        while (toBrowse.length > 0){

```

Comment:
This made me realise that we have no code style configuration in the project. I've just pushed an editorConfig and an idea code-style xml document. Please conform your contributions to match.